packages/ui/src/components/search/SearchResult.tsx

```diff
@@ -0,0 +1,169 @@
+import type { APISearchResult } from '@answeroverflow/api';
+import {
+	Message,
+	MessageContents,
+	MessageImages,
+	MessageRenderer,
+	MessageTitle,
+} from '~ui/components/primitives/Message';
+import { Button } from '../primitives/Button';
+
+export interface MessageResultProps {
```

Comment:
Can we make this look more like the message component which uses context?

Reply:
Done